### PR TITLE
Added issue template for codebase suggestion

### PR DESCRIPTION
**Pull request by amitnovick**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/codebase_suggestion.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Codebase Suggestion
+description: Refactorings / Updating dependencies / About development tools
+labels: codebase suggestion
```
Comment (rolandlo) [inline .github/ISSUE_TEMPLATE/codebase_suggestion.yml:3]:
```suggestion
labels: "codebase suggestion"
```

File: .github/ISSUE_TEMPLATE/codebase_suggestion.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      required: true
+  - type: textarea
+    attributes:
+      label: Alternative Suggestions
+      description: Add any alternative suggestions that you may have considered here.
+      placeholder: Ex. I also thought about...
+  - type: textarea
+    attributes:
+      label: Additional Context
+      description: Add any other contextual information for the suggestion here.
```
Comment (rolandlo) [inline .github/ISSUE_TEMPLATE/codebase_suggestion.yml:25]: The body should be tailored for codebase suggestions. This looks like copy+paste from the feature request .yml.

**Review by rolandlo — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/codebase_suggestion.yml (hunk 2 above)
Comment (amitnovick) [reply .github/ISSUE_TEMPLATE/codebase_suggestion.yml:25]: It is copied from the feature request, slightly altered with wording.

Do you have a suggestion to have something different?

Comment (rolandlo) [reply .github/ISSUE_TEMPLATE/codebase_suggestion.yml:25]: I would ask for what refactorings are suggested, what dependencies changes are suggested, what is the motivation and who is supposed to do the job. The template should also make it clear somehow that it is a template for developers.

**Comment by rolandlo** [comment]
Closing due to inactivity. Feel free to reopen if you feel like working on the PR again.

**Pull request closed without merging**